index.php

```diff
@@ -19,9 +28,31 @@
     <script defer src="js/index.js"></script>
 </head>
 
-<body>
-
-<h1>Website Template</h1>
 
+<body>
+    <header>
+        <h1>Games Collection</h1>
+    </header>
+    <main>
+        <span class="game">
+            <?php
+
+            $gameModel = new GameModel($db);
+            $games = $gameModel->getAllGames();
+
+            foreach ($games as $game) {
+                echo "<h2>$game->name</h2>";
+                echo "<p>Franchise: $game->franchise</p>";
+                echo "<p>Price: $game->price</p>";
+                echo "<p>Genre: $game->genre</p>";
+            }
```

Comment:
What if, for some reason, there are no games? Worth checking and displaying a simple message just in case